analysis.py

```diff
@@ -146,6 +151,29 @@ def gpgllParse(self, gpgll, line):
          if self.verifyChecksum(line, gpgll[5]):
              return gpgll
 
+
+ 	 '''
+
+    	 GPVTG properties
+    
+   	 gpvtg[0] - "Track Made Good and Ground Speed"
+  	 gpvtg[1] - True track made good
+   	 gpvtg[2] - Magnetic track made good
+   	 gpvtg[3] - Ground speed, knots
+   	 gpvtg[4] - ground speed, Kimometers per hour
+   	 gpvtg[5] - Checksum
+
+   	 '''
+
+    def gpvtgParse(self, gpvtg, line):
+         gpvtg[1] = line[1] + line[2]
+         gpvtg[2] = ("0.00" if line[3] == "" else line[3])+line[4]
```

Comment:
Can you add a comment above Line 170 clarifying why we need to check that one specifically for empty string?
Something like, "The data we receive for this field can be empty, so we check for it being empty here and insert 0.00 instead"